transactions/btc.utils.ts

```diff
@@ -6,6 +6,7 @@ import { createTransaction } from './btc';
 
 // we import the file into itself to enable mocking of recursive functions in the tests
 import * as self from './btc.utils';
+import { getOrdinalsByAddress } from 'api';
```

Comment:
is the import correct here? or do we have an alias for api?

Reply:
we have alias